taxcalc/current_law_policy.json

```diff
@@ -3093,6 +3093,50 @@
         "out_of_range_action": "stop"
     },
 
+    "_PT_exclusion_rt": {
+        "long_name": "Pass-through income exclusion rate",
+        "description": "Fraction of qualified business income excluded from taxable income.",
+        "section_1": "Personal Income",
+        "section_2": "Pass-Through",
+        "irs_ref": "",
+        "notes": "Applies to e00900 + e26270",
+        "row_var": "FLPDYR",
+        "row_label": ["2013"],
+        "start_year": 2013,
+        "cpi_inflated": false,
+        "col_var": "",
+        "col_label": "",
+        "boolean_value": false,
+        "integer_value": false,
+        "value": [0.0],
+        "range": {"min": 0.0, "max": 1.0},
+        "out_of_range_minmsg": "",
+        "out_of_range_maxmsg": "",
+        "out_of_range_action": "stop"
+    },
+
+    "_PT_exclusion_limit": {
```

Comment:
Could this be _PT_exclusion_wage_limit ?

Reply:
That would be fine.